gate/gate-web/src/main/groovy/com/netflix/spinnaker/gate/Main.groovy

```diff
@@ -40,7 +40,7 @@ import com.netflix.spinnaker.kork.boot.DefaultPropertiesBuilder
 )
 class Main {
 
-  static final Map<String, String> DEFAULT_PROPS = new DefaultPropertiesBuilder().property("spring.application.name", "gate").build()
+  static final Map<String, String> DEFAULT_PROPS = new DefaultPropertiesBuilder().property("spring.application.name", "gate").property("spring.mvc.pathmatch.matching-strategy","ANT_PATH_MATCHER").build()
```

Comment:
I hesitate to add more stuff here, when we've got gate-web/config/gate.yml.  Thoughts about moving this there?